### PR TITLE
Aha/export job part

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/tenants/mongo/MongoUserAccountExportHandler.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package sirius.biz.tenants.mongo;
+
+public class MongoUserAccountExportHandler {
```
Comment (tbiScireum) [inline src/main/java/sirius/biz/tenants/mongo/MongoUserAccountExportHandler.java]: empty class?

File: src/main/java/sirius/biz/jobs/batch/file/EntityExportJob.java [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+        AtomicBoolean seenTemplateRow = new AtomicBoolean(false);
+        LineBasedProcessor.create(templateFile.name(), templateFile.createInputStream()).run((rowNumber, row) -> {
+            if (row.length() == 0) {
+                return;
+            }
+
+            if (!dictionary.hasMappings()) {
+                determineMappingsFromRow(rowNumber, row);
+            } else {
+                if (!seenTemplateRow.get()) {
```
Comment (sabieber) [inline src/main/java/sirius/biz/jobs/batch/file/EntityExportJob.java]: Monoflop?

File: src/main/java/sirius/biz/jobs/batch/file/EntityExportJob.java [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+                                     .to(Log.BACKGROUND)
+                                     .error(e)
+                                     .withNLSKey("LineBasedJob.failureInRow")
+                                     .set("row", rowNumber)
+                                     .handle());
+        }
+    }
+
+    /**
+     * Creates an extractor for each known column if the effective mapping.
```
Comment (sabieber) [inline src/main/java/sirius/biz/jobs/batch/file/EntityExportJob.java]:
```suggestion
     * Creates an extractor for each known column of the effective mapping.
```

File: src/main/resources/biz_de.properties [hunk 4]
```diff
@@ -547,6 +547,7 @@ FieldDefinition.name = Name
 FieldDefinition.type = Typ
 FieldDefinition.remarks = Bemerkungen
 ImportDictionary.automaticMappings = Die Spaltenzuordnungen werden automatisch anhand der ersten Zeile und der Feldnamen (bzw. alternativen Bezeichnungen) ermittelt. Hierbei werden Groß/Kleinschreibung sowie Leerzeichen ignoriert.
+EntityExportJobFactory.templateModes = Wenn eine Vorlage vorliegt, werden die Spaltenzuordnungen automatisch anhand der ersten Zeile und der Feldnamen (bzw. alternativen Bezeichnungen) ermittelt. Hierbei werden Groß/Kleinschreibung sowie Leerzeichen ignoriert. Wenn die Vorlage weitere Zeilen enthält, so werden diese (wenn möglich) angereichert anstatt alle Datensätze auszugeben.
```
Comment (sabieber) [inline src/main/resources/biz_de.properties]:
```suggestion
EntityExportJobFactory.templateModes = Wenn eine Vorlage vorliegt, werden die Spaltenzuordnungen automatisch anhand der ersten Zeile und der Feldnamen (bzw. alternativen Bezeichnungen) ermittelt. Hierbei werden Groß-/Kleinschreibung sowie Leerzeichen ignoriert. Wenn die Vorlage weitere Zeilen enthält, so werden diese (wenn möglich) angereichert statt alle Datensätze auszugeben.
```

**Pull request merged**